native/cocos/editor-support/MeshBuffer.cpp

```diff
@@ -24,9 +24,23 @@
 ****************************************************************************/
 
 #include "MeshBuffer.h"
+#include "2d/renderer/UIMeshBuffer.h"
 
 MIDDLEWARE_BEGIN
 
+static const ccstd::vector<gfx::Attribute> ATTRIBUTES_V3F_T2F_C4B{
+    gfx::Attribute{gfx::ATTR_NAME_POSITION, gfx::Format::RGB32F},
+    gfx::Attribute{gfx::ATTR_NAME_TEX_COORD, gfx::Format::RG32F},
+    gfx::Attribute{gfx::ATTR_NAME_COLOR, gfx::Format::RGBA8, true},
+};
+
+static const ccstd::vector<gfx::Attribute> ATTRIBUTES_V3F_T2F_C4B_C4B{
+    gfx::Attribute{gfx::ATTR_NAME_POSITION, gfx::Format::RGB32F},
+    gfx::Attribute{gfx::ATTR_NAME_TEX_COORD, gfx::Format::RG32F},
+    gfx::Attribute{gfx::ATTR_NAME_COLOR, gfx::Format::RGBA8, true},
+    gfx::Attribute{gfx::ATTR_NAME_COLOR2, gfx::Format::RGBA8, true},
+};
+
```

Comment:
Should we put Attribute  in the same place？

Reply:
It is better in define in anonymous namespace.